### PR TITLE
CLOUDP-272416: Remove datalake tests

### PR DESCRIPTION
**Pull request by blva**
## Proposed changes

_Jira ticket:_ CLOUDP-272416

## Checklist

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by gssbzn — COMMENTED** [review]
Let's do the deprecation please, specially since we'll release this week

**Review by jeroenvervaeke — DISMISSED** [review]

**Comment by blva** [comment]
also: currently

```shell
➜  mongodb-atlas-cli git:(master) ./bin/atlas datalake
Command "dataLakes" is deprecated, Please use 'atlas datafederation'
The datalakes command provides access to your project data lakes. You can create, edit, and delete data lakes.

Usage:
```

**Comment by gssbzn** [comment]
>/bin/atlas datalake

that's datalake, you need to deprecate `atlas dataLakePipelines`

**Review by gssbzn — DISMISSED** [review]
LGTM

File: internal/cli/datalakepipelines/data_lake_pipelines.go [hunk 1]
```diff
@@ -35,9 +35,10 @@ func autoCompleteSourceType(_ *cobra.Command, _ []string, _ string) ([]string, c
 func Builder() *cobra.Command {
 	const use = "dataLakePipelines"
 	cmd := &cobra.Command{
-		Use:     use,
-		Aliases: cli.GenerateAliases(use),
-		Short:   "Data Lake pipelines.",
+		Use:        use,
+		Aliases:    cli.GenerateAliases(use),
+		Short:      "Data Lake pipelines.",
+		Deprecated: "Data Lake Pipelines is deprecated. Please see: https://www.mongodb.com/docs/atlas/app-services/data-api/data-api-deprecation.",
```
Comment (gssbzn) [inline internal/cli/datalakepipelines/data_lake_pipelines.go:41]:
```suggestion
		Deprecated: "Data Lake Pipelines is deprecated. Please see: https://dochub.mongodb.org/core/data-lake-deprecation.",
```

**Review by gssbzn — APPROVED** [review]

**Review by pierwill — APPROVED** [review]

**Pull request merged**